cmd/ocitools/generate.go

```diff
@@ -83,19 +87,30 @@ var generateCommand = cli.Command{
 			return err
 		}
 
+		specgen.ExportOpts.Seccomp = context.Bool("seccomp-only")
+
 		if context.IsSet("output") {
-			output := context.String("output")
-			err = specgen.SaveToFile(output)
+			specgen.ExportOpts.Output = context.String("output")
 		} else {
-			err = specgen.Save(os.Stdout)
+			specgen.ExportOpts.Output = "STDOUT"
```

Comment:
I'd rather not have this magic string.

And the library API already has `Save` and `SaveToFile`.  I think keeping the current switch here is fine, in which case you can drop `exportConfig`.